packages/strapi-plugin-i18n/admin/src/utils/getDefaultLocale.js

```diff
@@ -0,0 +1,58 @@
+const hasLocalePermission = (permission, localeCode) => {
```

Comment:
`const hasLocalePermission = (permissions, localeCode) => {`

Reply:
It's a unique permission here 😊